src/templates/finops-hub/modules/dataFactory.bicep

```diff
@@ -1863,8 +1698,7 @@ resource pipeline_ToIngestion 'Microsoft.DataFactory/factories/pipelines@2018-06
         typeProperties: {
           variableName: 'metric'
           value: {
-            // TODO: Parse metric out of the manifest file @ msexports/<scope>/<export-name>/<date-range>/[<timestamp?>/]<guid>/manifest.json
-            value: 'focuscost'
+            value: '@if(contains(toLower(pipeline().parameters.blobPath), \'amortizedcost\'), \'amortizedcost\', if(contains(toLower(pipeline().parameters.blobPath), \'actualcost\'), \'actualcost\', \'costdetails\'))'
```

Comment:
Can we make this passthru? I'd rather not change it so that way people know what dataset it is. Either that or we say that supported datasets get mapped to a known folder and unsupported datasets go to some other path? Also, should we include the data version as part of this? FOCUS has a new version coming next month.

Reply:
I'll put this in a new PR.